src/data/set/finite.lean

```diff
@@ -485,6 +485,9 @@ alias finite_univ_iff ↔ _root_.finite.of_finite_univ _
 theorem finite.union {s t : set α} (hs : s.finite) (ht : t.finite) : (s ∪ t).finite :=
 by { casesI hs, casesI ht, apply to_finite }
 
+theorem finite.union_compl {s : set α} (hs : s.finite) (hsc : sᶜ.finite) : finite α :=
```

Comment:
I'm not sure this is the right name. `finite.finite_of_compl`? (Theory: the `s.finite` is not mentioned since it's the receiver of the dot notation, and the result is a `finite`.)